src/components/NameListContext.js

```diff
@@ -29,14 +29,24 @@ class NameListProvider extends React.Component {
       preferences: [],
       sortBy: "name",
       namesPerPage: 25,
-
+      chartData: [],
+      filterDisplay: "visible",
+      // Functional Export
       onChangeOrder: this.onChangeOrder,
       onChangeNumber: this.onChangeNumber,
       onAdd: this.onAdd,
       onDelete: this.onDelete,
       callList: this.callList,
+      callChart: this.callChart,
```

Comment:
Changing this function's name to `createChart` will give readers a better idea of what calling it does.

Reply:
Sounds good / Done.